### ISSUE TITLE
information required - clear formatting

### ISSUE THREAD

**Issue by mikeatcityuni** [damyon/moodle-editor_atto#20]
Hi, is there any documentation on how 'clear formatting' button is meant to work in different browsers? We have a user who was expecting something similar to TinyMCE in terms of stripping MS Word tags but it doesn't seem to do very much.

**Comment by mikeatcityuni** [comment]
should add this is with Atto installed in M2.6 ... I gather there is different behaviour in 2.6 than 2.7+

**Comment by damyon** [comment]
Yep Atto in the plugins DB for 2.6 is completely different code to the one in core for 2.7 - but the clear formatting options works the same way in both - it's browser dependant but generally it only strips inline styles from the text. 

Atto in 2.7 always cleans up dodgy code from word - there is no button for it, it happens every time you save.
